Automattic/jetpack
# Jetpack: Check block dependencies after build

## Proposed changes:

Block view scripts should avoid large dependencies such as React. Add a script checking the `view.asset.php` files to ensure these aren't added accidentally.

### Other information:


## Jetpack product discussion


Fixes #28561

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Revert 87eb8515cd363e43852a624df3563b264ff247e7 and try building Jetpack. It should fail with appropriate messages.
* Try making various adjustments to the configuration arrays and try building.
